xml/System.Security.Cryptography.Pkcs/SignerInfo.xml

```diff
@@ -59,9 +59,13 @@
         <Parameter Name="asnEncodedData" Type="System.Security.Cryptography.AsnEncodedData" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="asnEncodedData">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="asnEncodedData">The attribute to add, an object with Abstract Syntax Notation One (ASN.1)-encoded data.</param>
+        <summary>Adds the specified attribute to the current instance's attribute collection. If the attribute already exists, it gets merged with the specified attribute. The document gets re-encoded after the addition.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.Security.Cryptography.CryptographicException">Cannot find the original signer.
```

Comment:
```suggestion
        <exception cref="T:System.Security.Cryptography.CryptographicException">Cannot find the original signer.
        
```

Reply:
What changed in this suggestion?

Reply:
There needs to be a blank line between the opening exception tag and the `-or-`.